Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/VaccinationCertificate.kt

```diff
@@ -1,20 +1,24 @@
 package de.rki.coronawarnapp.vaccination.core
 
 import de.rki.coronawarnapp.ui.Country
-import org.joda.time.Instant
 import org.joda.time.LocalDate
 
-data class VaccinationCertificate(
-    val firstName: String,
-    val lastName: String,
-    val dateOfBirth: LocalDate,
-    val vaccinatedAt: Instant,
-    val vaccineName: String,
-    val vaccineManufacturer: String,
-    val chargeId: String,
-    val certificateIssuer: String,
-    val certificateCountry: Country,
-    val certificateId: String,
-) {
-    val identifier: VaccinatedPersonIdentifier get() = ""
+interface VaccinationCertificate {
+    val firstName: String?
+    val lastName: String
+    val dateOfBirth: LocalDate
+    val vaccinatedAt: LocalDate
+
+    val vaccineName: String
+    val vaccineManufacturer: String
+    val medicalProductName: String
+
+    val doseNumber: Int
+    val totalSeriesOfDoses: Int
+
+    val certificateIssuer: String
+    val certificateCountry: Country
```

Comment:
Ist that correct? Shouldn't it be a String or ISO code?

Reply:
`VaccinationCertificate` is user facing, we internally work with strings, I initially thought that we might need to display some kind of country flag.

I'd put this up to @mtwalli  or @LukasLechnerDev  to refactor in a future PR if a string is more useful here.

Reply:
It was more about the value set we cover. We should accept all countries.